dandiapi/api/migrations/0001_initial.py

```diff
@@ -74,8 +75,8 @@ class Migration(migrations.Migration):
                     'blob',
                     models.FileField(
                         blank=True,
-                        storage=dandiapi.api.models.asset.get_asset_blob_storage,
-                        upload_to=dandiapi.api.models.asset.get_asset_blob_prefix,
+                        storage=dandiapi.api.storage.get_storage,
+                        upload_to=dandiapi.api.storage.get_storage_prefix,
```

Comment:
Why modify a past migration instead of making a new one? Will this cause problems with our heroku deployment?

Reply:
Oh I missed your comment on https://github.com/dandi/dandi-api/pull/640/commits/99464659b8d2f1601e2faf1f781faa7b6222ceb0, I think that answers my question.

Reply:
All the migration files are run in order when Django loads, so if they have import errors, the app crashes. Generally you shouldn't import anything in migrations to avoid exactly this problem. We ran into it because we wanted a handy helper function that generates storages.